test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlReaderTests.cs

```diff
@@ -2161,6 +2166,83 @@ public void ImportSchemaElementsOnlyFromReferencedNamespacesl()
             Assert.Null(personType);
         }
 
+        // We currently do not read the scale and srid values if the value is variable. 
+        [Fact]
+        public void ReadingLegacyScaleAndSridVariable_WithoutAnySetting_NotSupported()

```

Comment:
Do we support reading when `variable` is used? Do we support reading `Variable` when a setting is used?

Reply:
We do not support either.